gradle.properties

```diff
@@ -21,6 +21,8 @@ mockito.version=2.18.3
 guava.version=24.1-jre
 bouncycastle.version=1.59
 paho.version=1.2.0
+hivemq-testcontainer.version=2.0.0
+hivemq-extension-sdk.version=4.5.0
```

Comment:
why such an old version, or is there no newer out there?

Reply:
The problem is, that only 4.5.0 is compatible to a java 8 project like the `hivemq-mqtt-client`. Maybe we need to think about an other fix for this problem.